app/services/albums.js

```diff
@@ -2,30 +2,7 @@ const axios = require('axios');
 const config = require('../../config');
 
 exports.URL = config.common.urls.albums;
-exports.ALBUMS_SAMPLE = [
-  {
-    userId: 1,
-    id: 1,
-    title: 'quidem molestiae enim'
-  },
-  {
-    userId: 1,
-    id: 2,
-    title: 'sunt qui excepturi placeat culpa'
-  },
-  {
-    userId: 2,
-    id: 3,
-    title: 'omnis laborum odio'
-  },
-  {
-    userId: 2,
-    id: 4,
-    title: 'non esse culpa molestiae omnis sed optio'
-  }
-];
-
-exports.ALBUM_SAMPLE = id => exports.ALBUMS_SAMPLE[id - 1];
+exports.SAMPLES = require('./samples/album_samples');
```

Comment:
why dont you import it where you need it?

Reply:
its handy
at the same time you know that those samples corresponds to that service
plus you dont have requires everywhere